portfolio/src/main/webapp/gallery.html

```diff
@@ -7,16 +7,21 @@
   <link rel="stylesheet" href="style.css">
 </head>
 
-<body>
+  <body>
     <div id="content">
-        <h1 class="center"> Pictures of Jaden</h1>
-        <div class="img-format">
-          <img src="/Images/Jaden1.jpeg" alt="Image of Jaden" class="img">
-          <img src="/Images/Jaden2.jpeg" alt="Jaden Graduation"class="img">
-          <img src="/Images/Jaden3.jpeg" alt="Jaden in front of Bush Gardens christmas tree" class="img">
-          <img src="/Images/Jaden4.jpeg" alt="Jaden eating his first meal at hampton" class= "img">
-        </div>
+      <h1 class="center">Pictures of Jaden</h1>
+      <div class="img-format">
+        <img src="/Images/Jaden1.jpeg" title="Jaden's headshot" alt="Image of Jaden" class="img">
+        <img src="/Images/Jaden2.jpeg" title="Jaden at Graduation" alt="Jaden Graduation" class="img">
+        <img src="/Images/Jaden3.jpeg" title="Jaden at Bush Gardens" alt="Jaden in front of Bush Gardens christmas tree" class="img">
+        <img src="/Images/Jaden4.jpeg" title="Jaden eating at Hampton"alt="Jaden eating his first meal at hampton" class="img">
```

Comment:
Consistent spacing before "alt" attribute

Reply:
Got it.